src/com/yaya/sdk/Services/Transfer.java

```diff
@@ -0,0 +1,49 @@
+package com.yaya.sdk.Services;
+
+import com.yaya.sdk.ApiRequest.ApiClient;
+import com.yaya.sdk.Models.ExternalAccount;
+import com.yaya.sdk.Models.Fee;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.net.http.HttpResponse;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+public class Transfer {
+
+    private final ApiClient apiClient;
+    private final ObjectMapper objectMapper;
+
+    public Transfer(ApiClient apiClient) {
+        this.objectMapper = new ObjectMapper();
+        this.apiClient = apiClient;
+    }
+
+    public com.yaya.sdk.Models.Transfer[] getTransferList() throws IOException, InterruptedException, ExecutionException, NoSuchAlgorithmException, InvalidKeyException {
+        HttpResponse<String> response = apiClient.apiRequest("GET", "/transfer/", "", null);
```

Comment:
you can alternatively use `transfer/list` if you want to avoid the trailing slash